v2.0/sql-dump.md

```diff
@@ -14,8 +14,8 @@ The `cockroach dump` [command](cockroach-commands.html) outputs the SQL statemen
 
 When `cockroach dump` is executed:
 
-- Table and view schemas and table data are dumped as they appeared at the time that the command is started. Any changes after the command starts will not be included in the dump.
-- **New in v1.1:** Table and view schemas are dumped in the order in which they can successfully be recreated.
+- Table, sequence, and view schemas and table data are dumped as they appeared at the time that the command is started. Any changes after the command starts will not be included in the dump.
+- **New in v1.1:** Table, sequence, and view schemas are dumped in the order in which they can successfully be recreated.
```

Comment:
Sadly not true if you have cyclical foreign keys: https://github.com/cockroachdb/cockroach/issues/20254

Worth noting as a known limitation if it's not already.